### ISSUE TITLE
Wrap during search

### ISSUE THREAD

**Issue by GoogleCodeExporter** [snakefoot/snaketail-net#69]
```
I would like (optionally?) to be able to have the search feature wrap when it 
has reached the end of the file. I find I'm continually searching up when I 
mean down or vice versa. If I could set it so that it would wrap from the end 
back to the beginning, I would be able to find the text I'm looking for.


What version of the product are you using? On what operating system?
V 1.8.2.0 on Win 7

Thanks...
```

Original issue reported on code.google.com by `gsta...@gmail.com` on 14 Feb 2014 at 6:03

**Comment by snakefoot** [comment]
Thank you for the contribution. I have merged your commit to master, and tested. Please check if you like the result.

**Comment by snakefoot** [comment]
Closed with 00e12a0fcbbcf6998a107b72f3c78732e57d228d (Version 1.9.2)
